src/components/ReportList.js

```diff
@@ -1,10 +1,16 @@
 import React from "react";
 import Report from "./Report";
 
+function sortByContestId(a, b) {
+  return (
+    b.node.frontmatter.contest.contestid - a.node.frontmatter.contest.contestid
+  );
+}
+
 const ReportList = ({ reports }) => {
   return (
     <>
-      {reports.reverse().map((report) => (
+      {reports.sort(sortByContestId).map((report) => (
```

Comment:
Can we sort by start date instead? I imagine they're not in perfect order when using id since they could be created in a different order than they ran.